sansio_lsp_client/structs.py

```diff
@@ -192,6 +242,13 @@ class Location(BaseModel):
     range: Range
 
 
+class LocationLink(BaseModel):
+    originSelectionRange: t.Optional[Range]
+    targetUri: str  # DocumentUri...
```

Comment:
@halfbrained What does this comment mean?

Reply:
Just a note that in the specification this field is of type `DocumentUri`

Reply:
```suggestion
    targetUri: str  # in the spec the type is DocumentUri
```